src/stream/src/executor/source/fetch_executor.rs

```diff
@@ -80,6 +81,7 @@ impl<S: StateStore, Src: OpendalSource> FsFetchExecutor<S, Src> {
         source_ctx: SourceContext,
         source_desc: &SourceDesc,
         stream: &mut StreamReaderWithPause<BIASED, StreamChunk>,
+        rate_limit_rps: Option<u32>,
```

Comment:
What is `rps`? Why not just label it as `rate_limit_arg` or `rate_limit_org`.

Reply:
I think I've commented in the executor struct field, it means "row per sec". Motivation is to make the unit more visible, just like `timestamp_millis` or something.

Reply:
This is a minor issue. Even with docs, it still leads to readability issues IMO. But this won't block the PR from being merged.